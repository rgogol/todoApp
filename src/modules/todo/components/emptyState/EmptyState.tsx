import React from 'react';
import { View, Text } from 'react-native';
import EmptyStateStyles from './styles';

const EmptyState = () => {
  const { container, emptyTextStyle } = EmptyStateStyles;
  return (
    <View style={container}>
      <Text style={emptyTextStyle}>Type what you have to do</Text>
    </View>
  );
};

export default EmptyState;
