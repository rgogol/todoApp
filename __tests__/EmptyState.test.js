import React from 'react';
import { shallow } from 'enzyme';
import EmptyState from '../src/modules/todo/components/emptyState';

describe('EmptyState', () => {
  const wrapper = shallow(<EmptyState />).dive();

  it('render as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
