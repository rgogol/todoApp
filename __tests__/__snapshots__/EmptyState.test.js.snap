// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState render as expected 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 16,
      }
    }
  >
    Type what you have to do
  </Text>
</View>
`;
